src/lib/filesystem/src/Flow/Filesystem/Local/StdOutFilesystem.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\Filesystem\Local;
+
+use Flow\Filesystem\Exception\RuntimeException;
+use Flow\Filesystem\Local\StdOut\StdOutDestinationStream;
+use Flow\Filesystem\Path\Filter;
+use Flow\Filesystem\Path\Filter\KeepAll;
+use Flow\Filesystem\{DestinationStream, FileStatus, Filesystem, Path, Protocol, SourceStream};
+
+final class StdOutFilesystem implements Filesystem
+{
+    public function __construct(private readonly ?\php_user_filter $filter = null)
+    {
+    }
+
+    public function appendTo(Path $path) : DestinationStream
+    {
+        $this->protocol()->validateScheme($path);
+
+        return new StdOutDestinationStream($path, $this->filter);
+    }
+
+    public function getSystemTmpDir() : Path
+    {
+        throw new RuntimeException('StdOut does not have a system tmp directory');
```

Comment:
I'm considering splitting Filesystem interface into `ReadFilesystem` and `WriteFilesystem` interfaces to avoid breaking Liskov Substitution Principle